konvoy: Update docs for konvoy release v1.5.x-dk (#2836)
mesosphere/dcos-docs-site: pages/ksphere/konvoy/latest/install/install-gcp/index.md

---
layout: layout.pug
navigationTitle: Install on Google Cloud Platform (GCP)
title: Install on Google Cloud Platform (GCP)
menuWeight: 20
excerpt: Prepare for and install Konvoy on Google Cloud Platform (GCP)
enterprise: false
---

<!-- markdownlint-disable MD004 MD007 MD025 MD030 -->

> This section describes the steps to prepare your environment and install Konvoy on Google Cloud Platform (GCP).

Reviewer: ```suggestion
This section describes how to prepare your environment and install Konvoy on Google Cloud Platform (GCP).
```

## Prerequisites

* The [gcloud][install_gcloud] command line utility
* [Docker Desktop][install_docker] _version 18.09.2 or newer_
* [kubectl][install_kubectl] _v1.15.4 or newer_ (for interacting with the running cluster)
* A valid GCP account with [credentials configured][gcloud_sdk].

>   Make sure to set your `project` and `region` you want to use and export it

Reviewer: ```suggestion
  Make sure to set your `project` and `region` you want to use, and export it
```
